PaddleCV/PaddleDetection/docs/GETTING_STARTED.md

```diff
@@ -31,20 +74,53 @@ python tools/train.py -c configs/faster_rcnn_r50_1x.yml
 - RCNN models training on CPU is not supported on PaddlePaddle<=1.5.1 and will be fixed on later version.
```

Comment:
Model checkpoints is saved->Model checkpoints are saved

Reply:
DONE